### PR TITLE
Propose aws-lambda-nodejs-esbuild

### PR DESCRIPTION
**Pull request by floydspace**
Proposing to add `aws-lambda-nodejs-esbuild` construct lib.
Thank you

### CODE REVIEW

**Comment by kolomied** [comment]
Thank you for your contribution! 

While reviewing you library I noticed that `aws-lambda-nodejs-rollup` project (the one just above yours in the readme) got renamed to `aws-lambda-nodejs-webpack`. May I ask you to update name/link/description for that project as well as part of your PR? Ideally I'd like the descriptions to be identical except for the last bit (`webpack` vs `esbuild`).

I'm afraid I need to push for these additional changes to keep the list up to date.

Thank you again!

**Comment by floydspace** [comment]
@kolomied sure thing will do

**Comment by floydspace** [comment]
@kolomied, please verify

File: readme.md [hunk 1]
```diff
@@ -99,7 +99,8 @@ This section includes code libraries in various programming languages which vend
 
 * [AWS-CDK-Kotlin-DSL](https://github.com/justincase-jp/AWS-CDK-Kotlin-DSL) - A Wrapper library of [AWS CDK Java](https://mvnrepository.com/artifact/software.amazon.awscdk). CI automatically generates code and deploys it by daily.
 * [aws-cdk-maven-plugin](https://github.com/LinguaRobot/aws-cdk-maven-plugin) - A plugin to define and deploy your AWS CDK applications using Java and Maven.
-* [aws-lambda-nodejs-rollup](https://github.com/vvo/aws-lambda-nodejs-rollup) - Alternative Node.js lambda CDK construct, [using rollup.js](https://rollupjs.org/).
+* [aws-lambda-nodejs-webpack](https://github.com/vvo/aws-lambda-nodejs-webpack) - Alternative Node.js lambda CDK construct, [using webpack](https://webpack.js.org/).
+* [aws-lambda-nodejs-esbuild](https://github.com/floydspace/aws-lambda-nodejs-esbuild) - Alternative Node.js lambda CDK construct, [using esbuild](https://github.com/evanw/esbuild).
```
Comment (kolomied) [inline readme.md:103]: Can you please  change 

[using webpack](https://webpack.js.org/) 
to
using [webpack](https://webpack.js.org/)

And the same for `esbuild`. This is the last change, promise.

Comment (floydspace) [reply readme.md]: okay, did it

**Review by kolomied — APPROVED** [review]

**Comment by kolomied** [comment]
@floydspace  Please add "Mentioned in awesome" badge to your repo with the link to this list - [example](https://github.com/elegantdevelopment/aws-cdk-dynamodb-seeder).

Thanks for the contribution!

**Comment by floydspace** [comment]
@kolomied спасибо 👍

**Pull request merged**